Kirill Radov/game.js

```diff
@@ -0,0 +1,155 @@
+const stat = {
+    "statistic": [],
+    "statFileName": "stat.json",
+
+    statFile() {
+        if (!process.argv[2]) {
+            return this.statFileName;
+        } else {
+            return process.argv[2];
+        }
+    },
+
+    statisticWrite(userChoise, winOrLose, systemChoice) {
+       let stat = {"userChoice": userChoise,
```

Comment:
Рекомендую использовать const везде, где возможно
```suggestion
       const stat = {"userChoice": userChoise,
```